src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1307,7 +1307,7 @@ def _gethash(obj_info):
 
 
 # pylint: disable=missing-raises-doc
-def get_root(flproxy, version: str = "") -> Group:
+def get_root(flproxy, version: str = "23.2.0") -> Group:
```

Comment:
What happens if we keep `version` as an empty string `""` as it was before?

Is it only `tests\test_flobject.py` that is calling this without specifying the version? If that is the case then this should be fixed there in the tests, and not here

Reply:
Addressed by https://github.com/ansys/pyfluent/pull/2262/commits/d4e2d4c1aff4aa99a9d68e98442414360607305f